spec/features/guest/guest_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe Guest, type: :feature do
+  let(:hotel_1) { Hotel.create!(name: "Hilton", location: "Denver") }
+  let(:room_1) { Room.create!(rate: 100, suite: "The Blue Room", hotel_id: hotel_1.id) }
+  let(:guest_1) { Guest.create(name: "Bob", nights: 3) }
+  
+  #   As a visitor
+  # When I visit a guest's show page
+  # I see the guest's name
+  # And I see a list of all the rooms they've stayed in
+  # including the room's suite, nightly rate, and the name of the hotel that it belongs to.
+  
+  it "can see guest's name and a list of rooms they have stayed in" do
+    room_2 = Room.create!(rate: 200, suite: "The Red Room", hotel_id: hotel_1.id)
+    guest_1.rooms << room_1
+    guest_1.rooms << room_2
+    guest_1.save
```

Comment:
This `.save` is not required in a feature test; you can put it in a Controller though, if you're updating or creating a new instance of something.